sklearn/feature_selection/_rfe.py

```diff
@@ -71,19 +71,22 @@ class RFE(SelectorMixin, MetaEstimatorMixin, BaseEstimator):
 
     Attributes
     ----------
+    estimator_ : object
+        The external estimator fit on the reduced dataset.
+
+    classes_ : array-like, shape (n_classes,)
+        Unique class labels.
+
     n_features_ : int
         The number of selected features.
 
-    support_ : array of shape [n_features]
-        The mask of selected features.
-
     ranking_ : array of shape [n_features]
         The feature ranking, such that ``ranking_[i]`` corresponds to the
         ranking position of the i-th feature. Selected (i.e., estimated
         best) features are assigned rank 1.
 
-    estimator_ : object
-        The external estimator fit on the reduced dataset.
+    support_ : array of shape [n_features]
```

Comment:
`shape (n_features,)`

for consistency with the `(n_classes,)` above and general convention